scalability_and_performance/ztp-far-edge/ztp-upgrading-gitops.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: ASSEMBLY
+[id="ztp-upgrading-gitops"]
+= Upgrading GitOps ZTP
+include::_attributes/common-attributes.adoc[]
+:context: ztp-upgrading-gitops
+
+toc::[]
+
+You can upgrade the Gitops zero touch provisioning (ZTP) infrastructure independently from the underlying cluster, {rh-rhacm-first}, and {product-title} version running on the managed clusters.
```

Comment:
Does "underlying cluster" refer to the hub cluster? You could update that to be more explicit about what cluster is being referred to, given that there are several clusters in this infrastructure.